source/jormungandr/jormungandr/autocomplete/abstract_autocomplete.py

```diff
@@ -65,6 +74,12 @@ def status(self):
     def geo_status(self, instance):
         pass
 
+    def record_status(self, status, exc=None):
+        data = {'type': type(self).__name__, 'status': status}
```

Comment:
be careful with `type`
```python
class A(object):
    pass

class B:
    pass

print(type(A()).__name__)
print(type(B()).__name__)
```
```suggestion
         data = {'type': self.__class__.__name__, 'status': status}
```

Reply:
That's tricky :open_mouth: !

Reply:
good catch! :+1: